Annotation/GraphQLAccessControl.php

```diff
@@ -0,0 +1,17 @@
+<?php
+/**
+ * @author Thibault Colette <thibaultcolette06@hotmail.fr>
+ * @copyright 2018 Thibault Colette
+ */
+
+namespace Overblog\GraphQLBundle\Annotation;
+
+/**
+ * Annotation for graphql access control.
```

Comment:
Same as https://github.com/overblog/GraphQLBundle/pull/360#discussion_r206287596

Reply:
This one is used to protect the access, is like `access: "@=isAuthenticated()"` in yml. (use the method args to define wich one to use).

Reply:
See https://github.com/overblog/GraphQLBundle/pull/360#discussion_r206302243 :D